src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,95 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final int DEFAULT_INCREMENT_VALUE_OF_ARRAY = 2;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] table;
+    private int threshold;
+    private int size;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_INITIAL_CAPACITY];
+        threshold = (int)(table.length * DEFAULT_LOAD_FACTOR);
+    }
 
     @Override
     public void put(K key, V value) {
-
+        Node<K, V> newNode = new Node<>(key, value);
+        if (size + 1 == threshold) {
+            resize();
+        }
```

Comment:
We need change array capacity, when size >= threshold
```suggestion
        if (size >= threshold) {
            resize();
        }
```